manifests/4.5/elasticsearch-operator.v4.5.0.clusterserviceversion.yaml

```diff
@@ -123,9 +123,14 @@ spec:
         - apiGroups:
           - console.openshift.io
           resources:
+          - consolelinks
           - consoleexternalloglinks
           verbs:
-          - "*"
```

Comment:
functionally, is there a difference between the two? (is `patch` one of the verbs that's included with `*` ?)

Reply:
Patch and Watch are included in `*`. IMHO we should start do be more diligent on verb usage.

Reply:
auth team used to fuss at my PRs if I didn't explicitly enumerate verbs instead of `*`. Its prob a good idea! :)

Reply:
Do you need all the `verbs` on `consolelinks` and `consoleloglinks`?  I'd prob limit it down to what you need only.